pkg/peerhealth/server.go

```diff
@@ -111,7 +101,8 @@ func (s *Server) Start(ctx context.Context) error {
 }
 
 // IsHealthy checks if the given node is healthy
-func (s Server) IsHealthy(ctx context.Context, request *HealthRequest) (*HealthResponse, error) {
+func (s *Server) IsHealthy(ctx context.Context, request *HealthRequest) (*HealthResponse, error) {
+	s.log.Info("IsHealthy", "node", request.GetNodeName(), "machine", request.GetMachineName())
```

Comment:
do we want to keep this log?
If yes, we might improve it a little ("Another peer asking check", or something like that)

Reply:
yes I kept this on purpose, but good idea to improve the message